### ISSUE TITLE
Symfony 7?

### ISSUE THREAD

**Issue by treeindark** [craue/CraueGeoBundle#24]

**Comment by timrasche** [comment]
???

**Comment by pczjrh** [comment]
Yes, this needs addressing. You can not upgrade to Symfony7 if using this package:

Composer output reports:

  Problem 1
    - Root composer.json requires craue/geo-bundle ^1.9.0 -> satisfiable by craue/geo-bundle[1.9.0].
    - craue/geo-bundle 1.9.0 requires doctrine/orm ^2.5.2 -> found doctrine/orm[v2.5.2, ..., 2.20.1] but it conflicts with your root composer.json require (3.*).
Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.

Doctrine/ORM is on version >3.
